core/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
-from .views import (HackathonsRUDView,HackathonCreateView)
+from .views import HackathonTeamListView, HackathonTeamCreateView, HackathonListView, HackathonsRUDView, HackathonCreateView
 
-urlpatterns = [   
+urlpatterns = [
+    path('hackathons/<int:pk>/teams/', HackathonTeamListView.as_view()),
+    path('hackathons/add-team/', HackathonTeamCreateView.as_view()),
```

Comment:
This can be shifted to POST hackathons/<int:pk>/teams/

Reply:
@nishantwrp 
There is no RetrieveCreateAPIView to apply post and get in a generic view. And the post() method does not expect a 'pk' keyword. 
Should I move this to  ```POST /hackathons/teams/``` or is there any other way?